app/src/main/java/com/cornellappdev/scoop/ui/viewmodel/SearchScreenViewModel.kt

```diff
@@ -56,4 +56,7 @@ class SearchScreenViewModel @Inject constructor(private val searchRepository: Se
     fun setDepartureDate(date: String) {
         _search.departureDate = date
     }
+    fun filterRide(){
+
+    }
```

Comment:
Looks like u didn't run the linter 👀 , that `(){` spacing is sus... do this!

![image](https://github.com/cuappdev/scoop-android/assets/47724806/9bdb5a8f-0446-4760-85af-4c0f59f7d5c1)